test/test_interactive.py

```diff
@@ -82,6 +82,22 @@ def test_sdl_audio_mix_channels(self, args):
 
     self.btest_exit('test_sdl_audio_mix_channels.c', args=['-O2', '--minify=0', '--preload-file', 'sound.ogg'] + args)
 
+  @parameterized({
+    '': ([],),
+  })
+  def test_sdl_audio_mix_channels_halt(self, args):
+    shutil.copyfile(test_file('sounds/the_entertainer.ogg'), 'the_entertainer.ogg')
```

Comment:
Can you replace this line with `shutil.copy(test_file('sounds/the_entertainer.ogg'), '.')`?

Reply:
Does not work on windows:
```
Running the interactive tests. Make sure the browser allows popups from localhost.

test_sdl_audio_mix_playing (test_interactive.interactive.test_sdl_audio_mix_playing) ... ERROR
[Browser harness server terminated]

======================================================================
ERROR: test_sdl_audio_mix_playing (test_interactive.interactive.test_sdl_audio_mix_playing)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\caiiiycuk\sdk\emscripten\test\test_interactive.py", line 91, in test_sdl_audio_mix_playing
    shutil.copyfile(test_file('sounds/noise.ogg'), '.')
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.11_3.11.2544.0_x64__qbz5n2kfra8p0\Lib\shutil.py", line 258, in copyfile
    with open(dst, 'wb') as fdst:
         ^^^^^^^^^^^^^^^
PermissionError: [Errno 13] Permission denied: '.'

----------------------------------------------------------------------
Ran 1 test in 0.291s

FAILED (errors=1)
```

Reply:
Change `copyfile` to just `copy`